sig-release/charter.md

```diff
@@ -11,14 +11,15 @@ This charter adheres to the conventions described in the [Kubernetes Charter REA
 
 ### In scope
 
-- Ensuring quality Kubernetes releases
+- Declare a release and ensure quality/feature/other requirements are met.
   - Defining and staffing release roles to manage the resolution of release blocking criteria
   - Defining and driving development processes (e.g. merge queues, cherrypicks) and release processes
     (e.g. burndown meetings, cutting beta releases) with the intent of meeting the release schedule
-  - Managing the creation of release specific artifacts, including:
+  - Control access to and establish processes regarding release specific artifacts, including:
     - Code branches
     - Binary artifacts
     - Release notes
+    - Build Infrastructure
```

Comment:
Is this correct?

Reply:
I don't understand how the above wording improves things.  If anything, it *narrows* the scope of SIG-Release work.

Like, if we're not in charge of creating artifacts, who is?

Reply:
I view these as a super set of Creation.  e.g. does creation include update, delete, sharing, etc?

Reply:
We also have a few lines up in the doc "Production of Kubernetes releases on a reliable schedule".

I can see all of this meaning the same things and am not too bothered one way or the other.  That said...

Since the intent is moving words from steering's charter to release's charter, it might be easier to pass review without controversy if a single commit removed the phrases from steering's charter, added the exact removed phrases to release's charter, and made no other removals from release's charter.

In this way the observer sees a straight move.  And then SIG Release can carry on to its own discussion of whether and how we might de-duplicate and clarify our charter incrementally, but independent from the cleanup move.

Reply:
I think most of those words can be moved as bottom-level line items instead of redefining the general categories.

Reply:
ACK.  I actually don't think this change is that important.  Was just trying to cross t's and to i's.  I'll close the PR, and it can be updated in the future if needed.